### ISSUE TITLE
Cannot delete a column

### ISSUE THREAD

**Issue by corygehr** [gephi/gephi#1310]
In Data Table, clicking 'Delete column' and choosing the column to remove does not remove the column specified. The column remains in the data table.

**Comment by eduramiba** [comment]
Thanks for the report, this is fixed in graphstore (https://github.com/gephi/graphstore/issues/103) and will be included in next batch of updates and Gephi 0.9.1

**Comment by eduramiba** [comment]
Fixed with #1351

**Comment by eduramiba** [comment]
Actually a differention exception now occurs :cry: 

```java
java.lang.ArrayIndexOutOfBoundsException: 4
	at org.gephi.graph.impl.ColumnStore.removeColumn(ColumnStore.java:127)
	at org.gephi.graph.impl.TableImpl.removeColumn(TableImpl.java:124)
	at org.gephi.datalab.impl.AttributeColumnsControllerImpl.deleteAttributeColumn(AttributeColumnsControllerImpl.java:145)
	at org.gephi.datalab.plugin.manipulators.columns.DeleteColumn.execute(DeleteColumn.java:68)
[catch] at org.gephi.datalab.api.DataLaboratoryHelper$7.run(DataLaboratoryHelper.java:387)
```

Way to reproduce:
* Open les miserables
* Add a new node's column in datalab
* Delete the column

It does not happen if you delete the column having set the value for *all* nodes.

**Comment by mbastian** [comment]
This issue is fixed in graphstore, we shall update to 0.5.1 once its released

**Comment by eduramiba** [comment]
Thanks!
